### PR TITLE
New release

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
* Fitness-APP : refactoring info-panel by @RuffaloLavoisier in #278
* Add OSW-APP for Calendar by @RuffaloLavoisier in #279
* HAL: refactoring slice-weekday by @RuffaloLavoisier in #280
* OswAppWaterLevel: rename water-level by @RuffaloLavoisier in #281
* Set AP-Password by @RuffaloLavoisier in #282
* OswAppBLEMediaCtrl: typo in file by @RuffaloLavoisier in #283
* Fix list of date possibilities by @criscrafter in #286
* Emulator: add web interface virtual mode by @RuffaloLavoisier in #287
* Add Monotimer Watchface by @daschr in #288
* Implemented emulator for #190 by @simonmicro in #289
---
There was a change in the core part with the addition of the emulator. We've learned the lesson of doing a massive rebase and having to synchronize the branch periodically. (@simonmicro 👍 )

### CODE REVIEW

**Comment by simonmicro** [comment]
I'll review this later this week... Not today :)

**Comment by RuffaloLavoisier** [comment]
Have a good vacation:)

File: emulator/src/Display.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#include "../include/Display.h"
+
+std::unique_ptr<FakeDisplay> fakeDisplayInstance;
+
+FakeDisplay::FakeDisplay(int width, int height, SDL_Window* window, SDL_Renderer* renderer) : Arduino_G(width, height), mainWindow(window), mainRenderer(renderer), width(width), height(height) { };
+
+FakeDisplay::~FakeDisplay() {
+
+};
```
Comment (simonmicro) [inline emulator/src/Display.cpp:9]: We should remove the trailing `;`in the source files. These are for heady only :)

File: src/animations/anim_matrix.cpp [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "animations/anim_matrix.h"
```
Comment (simonmicro) [inline src/animations/anim_matrix.cpp:1]: Why does this file even exist?

**Review by simonmicro — APPROVED** [review]

**Pull request merged**